jupyter_server/event_schemas/kernel_actions/v1.yaml

```diff
@@ -69,3 +65,12 @@ properties:
     type: string
     description: |
       Description of the event specified in action.
+if:
```

Comment:
DId [this suggestion](https://github.com/jupyter-server/jupyter_server/pull/1300#issuecomment-1627395230) not work?  It just feels a bit more natural for the `if` portion to not be empty, wheres this code uses an "null" if and "non-null" else.

Reply:
Thanks Kevin. I will give it a try and update you.

Reply:
Tried your previous suggestion, it worked fine. Using that now.